tests/bud.bats

```diff
@@ -1821,3 +1821,11 @@ load helpers
 
   rm -f ${TESTSDIR}/bud/${target}/test*
 }
+
+@test "bud quite" {
+  run_buildah bud --format docker -t quite-test --signature-policy ${TESTSDIR}/policy.json -q ${TESTSDIR}/bud/shell
+  expect_line_count 1
```

Comment:
What if the line is a warning, or something unexpected? Please also include an `expect_output`.